### PR TITLE
Fix TOC for headlines with span

### PR DESCRIPTION
**Pull request by JanSchulz**
Closes: https://github.com/ipython-contrib/IPython-notebook-extensions/issues/380

### CODE REVIEW

File: nbextensions/usability/toc/main.js [hunk 1]
```diff
@@ -6,9 +6,9 @@ define(["require", "jquery", "base/js/namespace"], function (require, $, IPython
   var make_link = function (h) {
     var a = $("<a/>");
     a.attr("href", '#' + h.attr('id'));
-    // get the text *excluding* the link text, whatever it may be
     var hclone = h.clone();
-    hclone.children().remove();
+    // last() is the |P anchor which lights up when you hoover over the headline
+    hclone.children().last().remove();
     a.text(hclone.text());
```
Comment (jfbercher) [inline nbextensions/usability/toc/main.js]: Hi Jan, I think that there is perhaps a little better to do: try to replace the two .text() line 12 by .html(). This seems to enable to also render formatting in the link.

Comment (JanSchulz) [inline nbextensions/usability/toc/main.js]: Hm, at least `# Test $1=2$` is horrible with the current system... lets try your approach...

Comment (JanSchulz) [inline nbextensions/usability/toc/main.js]: Ok, text is much more readable...

**Comment by juhasch** [comment]
Looks good for me.

**Pull request merged**